### PR TITLE
Mobile user manual update: statistics

### PR DESCRIPTION
**Pull request by willemferguson**
As the title says. This is a first pass because I cannot see what it looks like
on the mobile device if it has not been pulled into master. I need to see what
size the images have on the mobile screen and how the organisation of text above
and below the images is rendered. There will definitely be a second PR to refine
the content for the mobile screen and to ensure proper integration of the
statistics section with the overall user manual.

Signed-off-by: willemferguson <willemferguson@zoology.up.ac.za>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [x] Documentation change
- [ ] Language translation

### Mentions:

@dirkhh @bstoeger

### CODE REVIEW

**Review by bstoeger — APPROVED** [review]
Fine with me - thanks.

File: Documentation/mobile-manual-v3.txt [hunk 1]
```diff
@@ -2,8 +2,8 @@
 // Subsurface-mobile User Manual
 // =============================
 // :author: Manual authors: Willem Ferguson, Dirk Hohndel
-// :revnumber: 3.0.0
-// :revdate: March 2020
+// :revnumber: 3.1.3
+// :revdate: February 2021
```
Comment (dirkhh) [inline Documentation/mobile-manual-v3.txt:6]: we'll make this v3.2

File: Documentation/mobile-manual-v3.txt [hunk 2]
```diff
@@ -491,7 +492,226 @@ The summary page lists the date of your first dive in the dive list as well as y
...
+so flexible that it is impossible to cover every possible use case. Rather, we provide a conceptual
+introduction, followed by specific examples. The tool can be closed using the _Back_ button (bottom
+right in Android).
+
+The Statistics tool has two panels (image on left):
+
+1. A setup panel containing comboboxes for requesting a graph (top).
+
+2. The graph that has been requested (bottom).
+
```
Comment (dirkhh) [inline Documentation/mobile-manual-v3.txt:516]: whether it's top/bottom or left/right depends on the screen orientation. But I guess as you refer to the screenshot this still makes sense

**Review by dirkhh — APPROVED** [review]
I really like this. Well done.
Smallish comments, but no reason not to merge so you can check image sizes, etc.

**Pull request merged**